### ISSUE TITLE
minor nitpick: this causes pyodide to also install mypy_extensions wheel when page is loaded

### ISSUE THREAD

**Issue by pgbovine** [SamLau95/pandas_tutor#184]
https://github.com/SamLau95/pandas_tutor/blob/e83b06642cdd4febe1711c3b6f30eb537dfbfbbf/setup.py#L20

(not sure whether this matters too much in the grand scheme of things since it's a small file, but i did notice that it tries to install it now when the page loads, probably due to this line)

**Comment by pgbovine** [comment]
note that i used to just inline these files inside the pandas_tutor repo in a bundled-dependencies.tar.gz file or something, but this is fine too for now, it seems to work

**Comment by SamLau95** [comment]
yeah, i tried to inline that package too but it caused problems in development. mypy complained that mypy-extensions shadows the built-in package and doesn't let me run typechecking :(

**Comment by SamLau95** [comment]
it looks like having `typing_extensions` in the root folder also crashes mypy in development
